hardly/handlers/distgit.py

```diff
@@ -100,6 +101,9 @@ def packit(self) -> PackitAPI:
                 ref=self.data.commit_sha,
                 working_dir=self.service_config.command_handler_work_dir,
             )
+            # We need to fetch tags from the upstream source-git repo
+            # Details: https://github.com/packit/hardly/issues/61
+            local_project.fetch(self.target_project_url)
```

Comment:
I'm not against, if you think it's more explicit, but one can get the project URL as:
```suggestion
            local_project.fetch(self.project.get_web_url())
```

Reply:
nice, I didn't know the info is in there, I thought that would be the fork